dashboard/app/controllers/resources_controller.rb

```diff
@@ -1,6 +1,32 @@
 class ResourcesController < ApplicationController
+  before_action :require_levelbuilder_mode, except: [:index, :show]
+
   # GET /resourcesearch/:q/:limit
   def search
     render json: ResourcesAutocomplete.get_search_matches(params[:q], params[:limit])
   end
+
+  def create
+    resource = Resource.new(
```

Comment:
I think at some point we will want to specify the course version here. one quirk I just discovered is that there might not always be a course version -- there is only one if the script is either in a unit group, or is marked as `is_course`. so one idea would be to start setting it here on a best-effort basis (i.e. when one is present). 

this makes me realize that we are going to need to be careful about setting up the 21-22 courses so that we do not accidentally change / lose the course version id since that could result in losing any lesson-resource associations.

Reply:
Yeah, we'll need to add the course version here at some point. I couldn't think of a simple way to do that in this PR so I decided to leave it for follow up.